pages/en/lb4/Creating-components.md

```diff
@@ -228,6 +230,33 @@ export class AuthenticationProvider {
 }
 ```
 
+## Extends Application
+
+When bind a component to an app, you may want to mount the component's propertis 
+and methods to application level context. 
+A stragety to do that is using mixin, which extends a class with new properties and methods.
```

Comment:
- typo: stragety --> strategy
- Not sure if i understand what it means..  Is it sth like:
a strategy is to use mixin to extends a class with new properties and methods ?

Reply:
@dhmlau yep.